polaris-react/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -62,6 +62,10 @@ export function hasGroupsWithActions(groups: ActionMenuProps['groups'] = []) {
 function convertGroupToSection({
   title,
   actions,
+  disabled,
```

Comment:
may be make it get a default value set to `false` to be backward compatible ? (assuming this is the public interface of the component)